pinot-query-planner/src/main/java/org/apache/pinot/query/parser/CalciteExpressionParser.java

```diff
@@ -46,10 +45,10 @@
 
 
 /**
- * Calcite parser to convert SQL expressions into {@link Expression}.
+ * This class provide API to parse a SQL string into Pinot query {@link SqlNode}.
```

Comment:
I feel this class should ideally be deleted. It will get difficult to maintain parity between this and existing CalciteSqlParser especially because both work on SQL -> SqlNode -> PinotQuery

Reply:
sounds good. it is not used.